tests/test_demes.py

```diff
@@ -11,7 +12,6 @@
     Builder,
     Epoch,
     AsymmetricMigration,
```

Comment:
If we only have AsymmetricMigration as the internal continuous migration class, should we call it simply Migration? Or leave it as AsymmetricMigration so that we leave open the option of reintroducing SymmetricMigration?

Reply:
Yeah, good question. I left it as `AsymmetricMigration` initially because it was easiest at the first stage of refactoring. But having this be explicit might help reduce confusion (at the expense of verbosity). And as you say, its not implausible that we'd introduce something called SymmetricMigration again in the future. But I don't have a strong opinion on what this should be called.

Reply:
More of a question than suggestion that we should change it. I'm happy to keep it as AsymmetricMigration, as it does reduce any possible confusion (and also allows us to introduce a SymmetricMigration if we are compelled to in the future again). Most users will still just see the `add_migration` function in the Builder, and won't even know how it is being represented behind the scene, I imagine.